build.sh

```diff
@@ -894,8 +894,13 @@ fi
 
 # Set default clang version
 if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
-    __ClangMajorVersion=3
-    __ClangMinorVersion=9
+	if [[ "$__BuildArch" == "arm" || "$__BuildArch" == "armel" || "$__BuildArch" == "arm64" ]]; then
```

Comment:
Do we want clang5.0 for arm64?

Reply:
I believe #11318 says that the problem affects arm64, but because we did not change docker for 16.04-cross-arm64 we can't update it now.

Reply:
Fixed, thanks.